news/fix-rattler-build-test-strict-parsing.rst

```diff
@@ -0,0 +1,3 @@
+**Fixed:**
+
+* Fix a test that uses `match` in skip to work with strict parsing (#2213)
```

Comment:
This looks like a left-over or C&P error

Reply:
Yep just force pushed, sry!

Reply:
Ah, this just disappeared with the force-push.